docs/geedocs/docsrc/answer/fusionTutorial/buildResource.rst

```diff
@@ -36,7 +36,7 @@ Defining and Building Resources
          -  :ref:`Define a Vector Resource <Define_a_Vector_Resource>`
          -  :ref:`Build a Vector Resource <Build_Vector_Resource>`
          -  :ref:`Define and Build the Remaining Vector
-            Resources <Define_Build_Remaining_Imagery_Resources>`
+            Resources <define-and-build-the-remaining-vector-resources>`
```

Comment:
Does this link format work? I've been adding tags above rubrics in the format:
 .. _Define_Build_Remaining_Vector_Resources:
and then using <Define_Build_Remaining_Vector_Resources> for the link reference

Reply:
It worked. I saw two different styles being used. We need to decide and stick to one